src/components/PrimaryNav/_primary-nav.scss

```diff
@@ -0,0 +1,57 @@
+@import "vanilla-framework/scss/settings_colors";
+
+.p-primary-nav {
+  a,
+  a:hover {
+    color: $color-light;
+  }
+
+  &__header {
+    display: grid;
+    grid-template-columns: 1fr 40px;
+    margin-bottom: -1rem;
+    padding: 0.5rem;
+  }
+
+  &__toggle {
+    background: none;
+    border: 0;
+    transition: transform 0.25s ease-in-out;
+
+    &:hover {
+      background: 0;
+    }
+  }
+
+  .p-list__link {
+    color: $color-light;
+  }
+
+  .p-list__item {
+    padding-left: 2.5rem;
+
+    &.is-selected {
+      background-color: #044f66;
+      border-left: 3px solid #3fc8f2;
+      margin-right: -0.5rem;
+      padding-left: calc(2.5rem - 3px);
+    }
+  }
+
+  .external {
+    margin-bottom: 0.25rem;
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.25s ease-in-out;
+  }
+
+  &.ext-nav-open {
+    .external {
+      max-height: 200px;
+    }
+
+    .p-primary-nav__toggle {
+      transform: rotate(180deg);
```

Comment:
In Chrome, clicking on the chevron 'selects' the element and because it's selected it also rotates the blue 'selection' border.

Firefox however has no such border and instead has a 'mouse down' background highlight which disappears when the animation starts. If you navigate to the element in FF and interact with it via the keyboard the blue border appears and rotates with the chevron.

In Chrome can we mirror the Firefox experience somehow? It's much cleaner. 👍

Reply:
You're talking about a browser level accessibility difference - something which it feels icky to mess with and even if we did, I'd prefer it was beyond the scope of this PR - please add a follow-up.